### PR TITLE
ER-651 - Moving away from open redirect 

### PR DESCRIPTION
**Pull request by sachpatel**
This is an enhancement to the item I did previously. Performing an open redirect based on the incoming `Referer` is bad practice as the referer could be an external site.

Happy to make changes if someone has a more appropriate solution.

### CODE REVIEW

**Comment by sachpatel** [comment]
@leewadhams Yes it should but I think this is a more clear way of performing the required redirect and it is a bit more explicit from within the controller code. I am happy to make any amendments or improvements if suggested?

**Comment by leewadhams** [comment]
👍 Was just checking that we do have the safety net of the NWebSec check.

File: src/Employer/Employer.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -90,7 +90,7 @@ public string GetTitle()
 
     <div asp-show="@ViewContext.ViewData[ViewDataKeys.CanShowOutageMessage] as bool?" id="global-outage-message" class="global-message">
         <p>@ViewContext.ViewData[ViewDataKeys.PlannedOutageMessage]</p>
-        <p><a asp-route="@RouteNames.DismissOutageMessage_Get">Dismiss</a></p>
+        <p><a asp-route="@RouteNames.DismissOutageMessage_Get" asp-route-returnUrl="@ViewContext.HttpContext.Request.Path">Dismiss</a></p>
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Views/Shared/_Layout.cshtml]: Could we encrypt the returnUrl? Then if it decrypts OK then we know its one of ours and don't need to do the regex.

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/Shared/_Layout.cshtml]: If we turn this to a POST then we'd have the CSRF security of the Anti-forgery token.

Comment (sachpatel) [reply src/Employer/Employer.Web/Views/Shared/_Layout.cshtml]: Will change to use POST 👍

File: src/Employer/Employer.Web/Startup.Configure.cs [hunk 2]
```diff
@@ -27,7 +27,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IOptions
             
             if (env.IsDevelopment())
             {
-                app.UseDeveloperExceptionPage();
+                //app.UseDeveloperExceptionPage();
+                app.UseExceptionHandler("/error/handle");
```
[the remark this answers is not in the document]
Comment (sachpatel) [reply src/Employer/Employer.Web/Startup.Configure.cs]: 😬 will remove.

**Pull request merged**